docs/index.asciidoc

```diff
@@ -87,6 +88,16 @@ filter {
   }
 }
 
+[id="plugins-{type}s-{plugin}-parser_options"]
+===== `parser_options`
+
+* Value type is <<string,string>>
+
```

Comment:
Consider adding default value here as a bullet, since it sounds like `strict` is not enabled by default.  I'm not sure how to notate that.  Would something like `* Default value is []` work?

Reply:
thanks, did the same as with rest ... to mention there's no default

Reply:
We documented `parser_options`, but the actual parameter is `parse_options` in the code. I can submit a PR to fix the doc but is it ok to keep `parse_options`?